runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ComputationState.java

```diff
@@ -62,6 +63,7 @@ public ComputationState(
     this.transformUserNameToStateFamily = ImmutableMap.copyOf(transformUserNameToStateFamily);
     this.executionStateQueue = new ConcurrentLinkedQueue<>();
     this.activeWorkState = ActiveWorkState.create(computationStateCache);
+    this.sourceBytesProcessCounter = "dataflow_source_bytes_processed-" + mapTask.getSystemName();
```

Comment:
how about sourceBytesProcessCounterName for var and method below?
soudns like this would be a counter object from var name

Reply:
done